openreview/venue/process/assignment_pre_process.js

```diff
@@ -0,0 +1,36 @@
+async function process(client, edge, invitation) {
+  client.throwErrors = true
+
+  const { groups } = await client.getGroups({ id: edge.domain })
+  const domain = groups[0]
+  const venue_id = domain.id
+  const submission_name = domain.content.submission_name.value
+  const review_name = invitation.content.review_name.value
+  const reviewers_anon_name = invitation.content.reviewers_anon_name.value
```

Comment:
In case some of these properties are not present in the content use optional chaining to avoid unintended errors.

```suggestion
  const submission_name = domain.content.submission_name?.value
  const review_name = invitation.content.review_name?.value
  const reviewers_anon_name = invitation.content.reviewers_anon_name?.value
```

Reply:
Thanks, I wonder if the variables should be named "submissionName", "reviewName", etc

Reply:
I would change them, but I don't think it's too bad.

Reply:
Done!